image_setup/01_base_images/install_rock_dependencies.bash

```diff
@@ -51,4 +51,7 @@ apt-get update && apt-get install -y \
  ruby \
  ruby-activesupport \
  ruby-dev \
- ruby-thor
+ ruby-thor \
+ libsdformat6-dev \
```

Comment:
I think we should stick to keeping the dependencies sorted in alphabetical order.